### PR TITLE
db: prepare DB upgrade to 5.38

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
db: prepare DB upgrade to 5.38

#### Ticket Link
N/A

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: store/sqlstore/upgrade.go [hunk 1]
```diff
@@ -1211,3 +1213,11 @@ func upgradeDatabaseToVersion537(sqlStore *SqlStore) {
 		saveSchemaVersion(sqlStore, Version5370)
 	}
 }
+
+func upgradeDatabaseToVersion538(sqlStore *SqlStore) {
+	// TODO: uncomment when the time arrive to upgrade the DB for 5.38
```
Comment (agnivade) [inline store/sqlstore/upgrade.go:1218]: Also, add note to bump `CurrentSchemaVersion`

**Review by agnivade — APPROVED** [review]
This may not be required by the time 5.38 gets released :) But anyways, good to have it in place.

Let's wait until Claudio comes back next week. There's no rush for this.

File: store/sqlstore/upgrade.go (hunk 1 above)
Comment (cpanato) [reply store/sqlstore/upgrade.go:1218]: yep, no rush :) thanks!

File: store/sqlstore/upgrade.go [hunk 2]
```diff
@@ -1211,3 +1213,11 @@ func upgradeDatabaseToVersion537(sqlStore *SqlStore) {
...
+	// if shouldPerformUpgrade(sqlStore, Version5370, Version5380) {
+
+	// 	saveSchemaVersion(sqlStore, Version5370)
```
Comment (streamer45) [inline store/sqlstore/upgrade.go:1221]:
```suggestion
	// 	saveSchemaVersion(sqlStore, Version5380)
```

**Review by streamer45 — APPROVED** [review]
Thanks, just left one suggestion :+1:

**Comment by cpanato** [comment]
@streamer45 @agnivade this is still needed? if not I will close, otherwise will fix the conflicts

**Comment by agnivade** [comment]
Hey @cpanato, yes this is needed.

**Comment by cpanato** [comment]
@agnivade i will check what happened, because I did not close this PR, I did the rebase and push

**Comment by cpanato** [comment]
looks like the changes is already there

**Pull request closed without merging**